extractors/python_functions/zipcode_extraction/code_snippet.md

```diff
@@ -1,12 +1,11 @@
 ```python
 import re
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes.
 YOUR_COUNTRY_IDS: List[str] = ["US"] # see list below for more countries
 YOUR_LABEL: str = "zip code"
 
-def zipcode_extraction(record) -> List[Tuple[str, int, int]]:
+def zipcode_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.